### ISSUE TITLE
Custom config_file not respected by scanner verification

### ISSUE THREAD

**Issue by chris-maclean** [kylefarris/clamscan#66]
When I initialize a NodeClam with a custom binary and config file for `clamdscan`, the scanner verification process does not respect the custom config file setting.

Here's my call to `init`:
```javascript
const clamscan = await new NodeClam().init({
    clamdscan: {
      path: '/opt/bin/clamdscan',
      config_file: '/opt/etc/myapp.conf',
      active: true,
    },
    preference: 'clamdscan',
    debug_mode: true,
  });
```

and here's the logger output:

```
node-clam: Could not verify the clamdscan binary. Error: Command failed: /opt/bin/clamdscan --version
ERROR: Can't parse clamd configuration file /opt/etc/clamd.conf

    at ChildProcess.exithandler (child_process.js:308:12)
    at ChildProcess.emit (events.js:314:20)
    at ChildProcess.EventEmitter.emit (domain.js:483:12)
    at maybeClose (internal/child_process.js:1022:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
    at Process.callbackTrampoline (internal/async_hooks.js:126:14) {
  killed: false,
  code: 2,
  signal: null,
  cmd: '/opt/bin/clamdscan --version',
  stdout: '',
  stderr: "ERROR: Can't parse clamd configuration file /opt/etc/clamd.conf\n"
}
```

As you can see, `clamdscan` is still looking for `/opt/etc/clamd.conf` (what I believe is the default config file location, and which doesn't exist in my environment) instead of the custom file I provided, `/opt/etc/myapp.conf`. The binary itself has been correctly overridden to the value I provided in `init`.

I believe the behavior comes from the `_is_clamav_binary` function in `index.js`

```javascript
async _is_clamav_binary(scanner) {
        const path = this.settings[scanner].path || null;
        if (!path) {
            if (this.settings.debug_mode) console.log(`${this.debug_label}: Could not determine path for clamav binary.`);
            return false;
        }

       // Right here
       // vvvvvvvvvvvv
        const version_cmds = {
            clamdscan: `${path} --version`,
            clamscan: `${path} --version`,
        };
       // ^^^^^^^^^^^^^^

        try {
            await fs_access(path, fs.constants.R_OK);
            const version_cmds_exec = version_cmds[scanner].split(' ');
            const {stdout} = await cp_execfile(version_cmds_exec[0], [version_cmds_exec[1]]);
            if (stdout.toString().match(/ClamAV/) === null) {
                if (this.settings.debug_mode) console.log(`${this.debug_label}: Could not verify the ${scanner} binary.`);
                return false;
            }
            return true;
        } catch (err) {
            if (this.settings.debug_mode) console.log(`${this.debug_label}: Could not verify the ${scanner} binary.`, err);
            return false;
        }
    }
```

The `version_cmds` commands do not include a `-c/--config-file` option, and my log output suggests that `clamdscan` still requires a working config file even just to run `--version`.

Can the package be modified to use the custom config file during scanner verification? I could take on this task if necessary.

**Comment by kylefarris** [comment]
Hey @chris-maclean, it looks like you've likely found the issue and have the perfect test bed to write up the PR for this issue. Please, if you have time, make the necessary changes and tests and I'll be more than happy to merge the PR.

Thanks for the bug report!

**Comment by kylefarris** [comment]
Fixed with #73
